// table-library/fesm2015/table-library.mjs
import * as i0 from '@angular/core';
import { Injectable, Component, NgModule } from '@angular/core';

class TableLibraryService {
    constructor() { }
}
TableLibraryService.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.7", ngImport: i0, type: TableLibraryService, deps: [], target: i0.ɵɵFactoryTarget.Injectable });
TableLibraryService.ɵprov = i0.ɵɵngDeclareInjectable({ minVersion: "12.0.0", version: "13.2.7", ngImport: i0, type: TableLibraryService, providedIn: 'root' });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.7", ngImport: i0, type: TableLibraryService, decorators: [{
            type: Injectable,
            args: [{
                    providedIn: 'root'
                }]
        }], ctorParameters: function () { return []; } });

class TableLibraryComponent {
    constructor() { }
    ngOnInit() {
    }
}
TableLibraryComponent.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.7", ngImport: i0, type: TableLibraryComponent, deps: [], target: i0.ɵɵFactoryTarget.Component });
TableLibraryComponent.ɵcmp = i0.ɵɵngDeclareComponent({ minVersion: "12.0.0", version: "13.2.7", type: TableLibraryComponent, selector: "lib-table-library", ngImport: i0, template: `
    <p>
      table-library works!
    </p>
  `, isInline: true });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.7", ngImport: i0, type: TableLibraryComponent, decorators: [{
            type: Component,
            args: [{
                    selector: 'lib-table-library',
                    template: `
    <p>
      table-library works!
    </p>
  `,
                    styles: []
                }]
        }], ctorParameters: function () { return []; } });

class HellowWorldComponent {
    constructor() { }
    ngOnInit() {
    }
}
HellowWorldComponent.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.7", ngImport: i0, type: HellowWorldComponent, deps: [], target: i0.ɵɵFactoryTarget.Component });
HellowWorldComponent.ɵcmp = i0.ɵɵngDeclareComponent({ minVersion: "12.0.0", version: "13.2.7", type: HellowWorldComponent, selector: "my-prefix-hellow-world", ngImport: i0, template: "<p>hellow-world works!</p>\n", styles: [""] });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.7", ngImport: i0, type: HellowWorldComponent, decorators: [{
            type: Component,
            args: [{ selector: 'my-prefix-hellow-world', template: "<p>hellow-world works!</p>\n", styles: [""] }]
        }], ctorParameters: function () { return []; } });

class TableLibraryModule {
}
TableLibraryModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.7", ngImport: i0, type: TableLibraryModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
TableLibraryModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.7", ngImport: i0, type: TableLibraryModule, declarations: [TableLibraryComponent,
        HellowWorldComponent], exports: [TableLibraryComponent, HellowWorldComponent] });
TableLibraryModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.7", ngImport: i0, type: TableLibraryModule, imports: [[]] });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.7", ngImport: i0, type: TableLibraryModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [
                        TableLibraryComponent,
                        HellowWorldComponent
                    ],
                    imports: [],
                    exports: [
                        TableLibraryComponent, HellowWorldComponent
                    ]
                }]
        }] });

/*
 * Public API Surface of table-library
 */

/**
 * Generated bundle index. Do not edit.
 */

export { HellowWorldComponent, TableLibraryComponent, TableLibraryModule, TableLibraryService };
