// table-library/esm2020/lib/table-library.module.mjs
import { NgModule } from '@angular/core';
import { TableLibraryComponent } from './table-library.component';
import { HellowWorldComponent } from './table-library/hellow-world/hellow-world.component';
import * as i0 from "@angular/core";
export class TableLibraryModule {
}
TableLibraryModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.7", ngImport: i0, type: TableLibraryModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
TableLibraryModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.7", ngImport: i0, type: TableLibraryModule, declarations: [TableLibraryComponent,
        HellowWorldComponent], exports: [TableLibraryComponent, HellowWorldComponent] });
TableLibraryModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.7", ngImport: i0, type: TableLibraryModule, imports: [[]] });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.7", ngImport: i0, type: TableLibraryModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [
                        TableLibraryComponent,
                        HellowWorldComponent
                    ],
                    imports: [],
                    exports: [
                        TableLibraryComponent, HellowWorldComponent
                    ]
                }]
        }] });
