// table-library/esm2020/lib/table-library.component.mjs
import { Component } from '@angular/core';
import * as i0 from "@angular/core";
export class TableLibraryComponent {
    constructor() { }
    ngOnInit() {
    }
}
TableLibraryComponent.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.7", ngImport: i0, type: TableLibraryComponent, deps: [], target: i0.ɵɵFactoryTarget.Component });
TableLibraryComponent.ɵcmp = i0.ɵɵngDeclareComponent({ minVersion: "12.0.0", version: "13.2.7", type: TableLibraryComponent, selector: "lib-table-library", ngImport: i0, template: `
    <p>
      table-library works!
    </p>
  `, isInline: true });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.7", ngImport: i0, type: TableLibraryComponent, decorators: [{
            type: Component,
            args: [{
                    selector: 'lib-table-library',
                    template: `
    <p>
      table-library works!
    </p>
  `,
                    styles: []
                }]
        }], ctorParameters: function () { return []; } });
